### PR TITLE
[NIXIO] Disable lazy load test

### PR DESCRIPTION
**Pull request by achilleas-k**
There seems to be an issue with the NIXRawIO that was introduced through PR #629 but only showed up when both that PR and #702 were merged.  I'm currently working to fix it, but as I understand the failing test is blocking other PRs from being merged.  I've disabled the test while I work on the core issue to keep other PRs from being merged.

### CODE REVIEW

**Comment by samuelgarcia** [comment]
@amchagas @JuliaSprenger we should merge this very soon.
So we could merge almost all PR for 0.8.
What do you think ?

File: neo/rawio/nixrawio.py [hunk 1]
```diff
@@ -359,23 +377,20 @@ def _rescale_epoch_duration(self, raw_duration, dtype='float64'):
...
+        Takes a collection of NIX metadata properties and the name of a Neo
+        type and returns a dictionary representing the Neo object annotations.
+        Properties that represent the attributes of the Neo object type are
+        filtered, based on the global 'neo_attributes' dictionary.
+        """
+        annotations = dict()
+        # ignore neo_name as well
+        attrs = neo_attributes.get(neo_type, list()) + ["neo_name"]
+        for prop in properties:
+            if not (prop.name in attrs or prop.name == "neo_name"):
```
Comment (JuliaSprenger) [inline neo/rawio/nixrawio.py]: isn't this handling `neo_name` twice? once in line 389 and once in line 391?

Comment (achilleas-k) [reply neo/rawio/nixrawio.py]: You're right.  Let me clean that up.

File: neo/rawio/nixrawio.py [hunk 2]
```diff
@@ -169,38 +178,47 @@ def _parse_header(self):
...
+                groupdas = NixIO._group_signals(grp.data_arrays)
+                for nix_name, signals in groupdas.items():
+                    da = signals[0]
+                    if da.type == 'neo.analogsignal' and seg_ann['signals']:
+                        # collect and group DataArrays
+                        sig_ann = seg_ann['signals'][sig_idx]
+                        # sig_chan_ann = self.raw_annotations['signal_channels'][sig_idx]
+                        props = da.metadata.inherited_properties()
+                        sig_ann.update(self._filter_properties(props, 'analogsignal'))
+                        # sig_chan_ann.update(self._filter_properties(props, 'analogsignal'))
```
Comment (JuliaSprenger) [inline neo/rawio/nixrawio.py]: do you stil need these commented code snippets?

Comment (achilleas-k) [reply neo/rawio/nixrawio.py]: They're part of the functions that need to be fixed, so it's a little bit in a WIP stage.  If you don't mind the cruft, I'd rather keep them for now.

**Pull request merged**